package/MDAnalysis/topology/__init__.py

```diff
@@ -100,6 +100,10 @@
                                  If atoms are written out of order, will correct the
                                  order.
                                  :mod:`MDAnalysis.topology.DLPolyParser`
+
+   Hoomd XML [#a]_   xml         HOOMD XML Topology file;
```

Comment:
So this xml can technically provide coordinates, but not until the corresponding Reader is written, so get rid of the [#a]_ for now